protocol-definitions/Topic.yaml

```diff
@@ -103,3 +104,83 @@ methods:
           since: 2.0
           doc: |
             True if registration is removed, false otherwise
+  - id: 4
+    name: publishAll
+    since: 2.0
```

Comment:
New additions should have `since: 2.1` .

Reply:
Ok. :)